feature_settings/src/main/java/com/my/kizzy/feature_settings/rpc_settings/RpcSettings.kt

```diff
@@ -407,5 +425,50 @@ fun RpcSettings(onBackPressed: () -> Boolean) {
                 }
             )
         }
+
+        if (showApplicationIdDialog) {
+            AlertDialog(
+                onDismissRequest = {
+                    showApplicationIdDialog = false
+                },
+                text = {
+                    Column {
+                        Text(text = stringResource(R.string.application_id))
+                        Spacer(modifier = Modifier.height(8.dp))
+                        TextField(
```

Comment:
Try to use `RpcField` component like how it's used above

Reply:
Looks like the RPCFIeld is a newly implemented composable?  I don't recall seeing this when I was working on it yesterday.

Reply:
The last commit should take care of this.